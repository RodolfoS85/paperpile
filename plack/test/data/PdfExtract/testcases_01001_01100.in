---
file: ../../../PDFREPOSITORY/001001.pdf
---
file: ../../../PDFREPOSITORY/001002.pdf
---
file: ../../../PDFREPOSITORY/001003.pdf
---
file: ../../../PDFREPOSITORY/001004.pdf
---
file: ../../../PDFREPOSITORY/001005.pdf
---
file: ../../../PDFREPOSITORY/001006.pdf
---
file: ../../../PDFREPOSITORY/001007.pdf
---
file: ../../../PDFREPOSITORY/001008.pdf
---
file: ../../../PDFREPOSITORY/001009.pdf
---
file: ../../../PDFREPOSITORY/001010.pdf
---
file: ../../../PDFREPOSITORY/001011.pdf
---
file: ../../../PDFREPOSITORY/001012.pdf
---
file: ../../../PDFREPOSITORY/001013.pdf
---
file: ../../../PDFREPOSITORY/001014.pdf
---
file: ../../../PDFREPOSITORY/001015.pdf
---
file: ../../../PDFREPOSITORY/001016.pdf
---
file: ../../../PDFREPOSITORY/001017.pdf
---
file: ../../../PDFREPOSITORY/001018.pdf
---
file: ../../../PDFREPOSITORY/001019.pdf
---
file: ../../../PDFREPOSITORY/001020.pdf
---
file: ../../../PDFREPOSITORY/001021.pdf
---
file: ../../../PDFREPOSITORY/001022.pdf
---
file: ../../../PDFREPOSITORY/001023.pdf
---
file: ../../../PDFREPOSITORY/001024.pdf
---
file: ../../../PDFREPOSITORY/001025.pdf
---
file: ../../../PDFREPOSITORY/001026.pdf
---
file: ../../../PDFREPOSITORY/001027.pdf
---
file: ../../../PDFREPOSITORY/001028.pdf
---
file: ../../../PDFREPOSITORY/001029.pdf
---
file: ../../../PDFREPOSITORY/001030.pdf
---
file: ../../../PDFREPOSITORY/001031.pdf
---
file: ../../../PDFREPOSITORY/001032.pdf
---
file: ../../../PDFREPOSITORY/001033.pdf
---
file: ../../../PDFREPOSITORY/001034.pdf
---
file: ../../../PDFREPOSITORY/001035.pdf
---
file: ../../../PDFREPOSITORY/001036.pdf
---
file: ../../../PDFREPOSITORY/001037.pdf
---
file: ../../../PDFREPOSITORY/001038.pdf
---
file: ../../../PDFREPOSITORY/001039.pdf
---
file: ../../../PDFREPOSITORY/001040.pdf
---
file: ../../../PDFREPOSITORY/001041.pdf
---
file: ../../../PDFREPOSITORY/001042.pdf
---
file: ../../../PDFREPOSITORY/001043.pdf
---
file: ../../../PDFREPOSITORY/001044.pdf
---
file: ../../../PDFREPOSITORY/001045.pdf
---
file: ../../../PDFREPOSITORY/001046.pdf
---
file: ../../../PDFREPOSITORY/001047.pdf
---
file: ../../../PDFREPOSITORY/001048.pdf
---
file: ../../../PDFREPOSITORY/001049.pdf
---
file: ../../../PDFREPOSITORY/001050.pdf
---
file: ../../../PDFREPOSITORY/001051.pdf
---
file: ../../../PDFREPOSITORY/001052.pdf
---
file: ../../../PDFREPOSITORY/001053.pdf
---
file: ../../../PDFREPOSITORY/001054.pdf
---
file: ../../../PDFREPOSITORY/001055.pdf
---
file: ../../../PDFREPOSITORY/001056.pdf
---
file: ../../../PDFREPOSITORY/001057.pdf
---
file: ../../../PDFREPOSITORY/001058.pdf
---
file: ../../../PDFREPOSITORY/001059.pdf
---
file: ../../../PDFREPOSITORY/001060.pdf
---
file: ../../../PDFREPOSITORY/001061.pdf
---
file: ../../../PDFREPOSITORY/001062.pdf
---
file: ../../../PDFREPOSITORY/001063.pdf
---
file: ../../../PDFREPOSITORY/001064.pdf
---
file: ../../../PDFREPOSITORY/001065.pdf
---
file: ../../../PDFREPOSITORY/001066.pdf
---
file: ../../../PDFREPOSITORY/001067.pdf
---
file: ../../../PDFREPOSITORY/001068.pdf
---
file: ../../../PDFREPOSITORY/001069.pdf
---
file: ../../../PDFREPOSITORY/001070.pdf
---
file: ../../../PDFREPOSITORY/001071.pdf
---
file: ../../../PDFREPOSITORY/001072.pdf
---
file: ../../../PDFREPOSITORY/001073.pdf
---
file: ../../../PDFREPOSITORY/001074.pdf
---
file: ../../../PDFREPOSITORY/001075.pdf
---
file: ../../../PDFREPOSITORY/001076.pdf
---
file: ../../../PDFREPOSITORY/001077.pdf
---
file: ../../../PDFREPOSITORY/001078.pdf
---
file: ../../../PDFREPOSITORY/001079.pdf
---
file: ../../../PDFREPOSITORY/001080.pdf
---
file: ../../../PDFREPOSITORY/001081.pdf
---
file: ../../../PDFREPOSITORY/001082.pdf
---
file: ../../../PDFREPOSITORY/001083.pdf
---
file: ../../../PDFREPOSITORY/001084.pdf
---
file: ../../../PDFREPOSITORY/001085.pdf
---
file: ../../../PDFREPOSITORY/001086.pdf
---
file: ../../../PDFREPOSITORY/001087.pdf
---
file: ../../../PDFREPOSITORY/001088.pdf
---
file: ../../../PDFREPOSITORY/001089.pdf
---
file: ../../../PDFREPOSITORY/001090.pdf
---
file: ../../../PDFREPOSITORY/001091.pdf
---
file: ../../../PDFREPOSITORY/001092.pdf
---
file: ../../../PDFREPOSITORY/001093.pdf
---
file: ../../../PDFREPOSITORY/001094.pdf
---
file: ../../../PDFREPOSITORY/001095.pdf
---
file: ../../../PDFREPOSITORY/001096.pdf
---
file: ../../../PDFREPOSITORY/001097.pdf
---
file: ../../../PDFREPOSITORY/001098.pdf
---
file: ../../../PDFREPOSITORY/001099.pdf
---
file: ../../../PDFREPOSITORY/001100.pdf
