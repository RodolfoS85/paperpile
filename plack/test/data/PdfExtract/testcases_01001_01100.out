---
authors: 'Noirel, Josselin and Simonson, Thomas'
doi: 10.1186/1472-6807-7-79
title: 'Neutral evolution of Protein-protein interactions: a computational study using simple models'
---
authors: 'Noirel, Josselin and Simonson, Thomas'
doi: 10.1063/1.2992853
title: 'Neutral evolution of proteins: The superfunnel in sequence space and its relation to mutational robustness'
---
authors: 'Nordeen, Steven K. and Ogden, Carol A. and Taraseviciene, Laima and Lieberman, Benjamin A.'
title: Extreme Position Dependence of a Canonical Hormone Response Element
---
authors: 'Notredame, Cédric and OBrien, Emmet A. and Higgins, Desmond G.'
title: 'RAGA: RNA sequence alignment by genetic algorithm'
---
authors: 'Novichkov, P. S. and Gelfand, M. S. and Mironov, A. A.'
title: Gene recognition in eukaryotic DNA by comparison of genomic sequences
---
authors: 'Nudler, Evgeny and Mironov, Alexander S.'
doi: 10.1016/j.tibs.2003.11.004
title: The riboswitch control of bacterial metabolism
---
authors: 'Nuin, Paulo AS and Wang, Zhouzhi and Tillier, Elisabeth RM'
doi: 10.1186/1471-2105-7-471
title: The accuracy of several multiple sequence alignment programs for proteins
---
authors: 'Oates, Tim and Heeringa, Brent'
title: Estimating Grammar Parameters using Bounded Memory
---
authors: 'Obernosterer, Gregor and Tafer, Hakim and Martinez, Javier'
doi: 10.1042/BST0361216
title: Target site effects in the RNA interference and microRNA pathways
---
authors: 'Oberstrass, Florian C. and Auweter, Sigrid D. and Erat,  and Hargous, Yann and Henning, Anke and Wenter, Philipp and Reymond, Luc and Amir-Ahmady, Batoul and Pitsch, Stefan and Black, Douglas L. and -T. Allain, Frederic H.'
title: 'Structure of PTB Bound to RNA: Specific Binding and Implications for Splicing Regulation'
---
authors: 'Odreman-Macchioli, F. E. and Tisminetzky, S. G. and Zotti, M. and Baralle, F. E. and Buratti, E.'
title: Influence of correct secondary and tertiary RNA folding on the binding of cellular factors to the HCV IRES
---
authors: 'Offman, Marc N and Nurtdinov, Ramil N and Gelfand, Mikhail S and Frishman, Dmitrij'
title: No statistical support for correlation between the positions of protein interaction sites and alternatively spliced regions
---
authors: 'Ogbourne, Steven and Antalis, Toni M.'
title: Transcriptional control and the role of silencers in transcriptional regulation in eukaryotes
---
authors: 'Ogihara, Mitsunori and Ray, Animesh'
title: DNA computing on a chip
---
authors: 'Ohler, Uwe and Liao, Guo-chun and Niemann, Heinrich and Rubin, Gerald M'
title: Computational analysis of core promoters in the Drosophila genome
---
authors: 'Ohler, Uwe and Shomron, Noam and Burge, Christopher B.'
doi: 10.1371/journal.pcbi.0010015
title: Recognition of Unknown Conserved Alternatively Spliced Exons
---
authors: 'Ohler, Uwe and Yekta, Soraya and Lim, Lee P. and Bartel, -DAVID P. and Burge, Christopher B.'
doi: 10.1261/rna.5206304
title: Patterns of flanking sequence conservation and a characteristic upstream motif for microRNA gene identification
---
authors: 'Ohto, Hiromi and Kamada, Sayaka and Tago, Kenji and Tominaga, SHIN-ICHI and Ozaki, Hidenori and Sato, Shigeru and Kawakami, Kiyoshi'
title: Cooperation of Six and Eya in Activation of Their Target Genes through Nuclear Translocation of Eya
---
arxivid: cond-mat/0308119
authors: 'Okamoto, Yuko'
title: Metropolis Algorithms in Generalized Ensemble
---
authors: 'Olman, Victor and Xu, Dong and Xu, Ying'
title: IDENTIFICATION OF REGULATORY BINDING SITES USING MINIMUM SPANNING TREES
---
authors: 'Olsnes, Sjur and Klingenberg, Olav and WIEDŁOCHA, Antoni'
doi: 10.1152/physrev.00021.2002
title: Transport of Exogenous Growth Factors and Cytokines to the Cytosol and to the Nucleus
---
authors: 'Orgel, L. E.'
title: Evolution of the Genetic Apparatus
---
authors: 'Ornitz, David M and Itoh, Nobuyuki'
title: Fibroblast growth factors
---
authors: 'Ornitz, David M. and Xu, Jingsong and Colvin, Jennifer S. and McEwen, Donald G. and MacArthur, Craig A. and Coulier, Francois and Gao, Guangxia and Goldfarb, Mitchell'
title: Receptor Specificity of the Fibroblast Growth Factor Family
---
authors: 'Ortiz, Angel R. and Kolinski, Andrzej and Skolnick, Jeffrey'
title: COMBINED MULTIPLE SEQUENCE REDUCED PROTEIN MODEL APPROACH TO PREDICT THE TERTIARY STRUCTURE OF SMALL PROTEINS
---
authors: 'Osada, Robert and Zaslavsky, Elena and Singh, Mona'
title: Comparative analysis of methods for representing and searching for transcription factor binding sites
---
authors: 'Ott, Carolyn M. and Lingappa, Vishwanath R.'
title: 'Integral membrane protein biosynthesis: why topology is hard to predict'
---
authors: 'Otto, Wolfgang and Will, Sebastian and Backofen, Rolf'
title: Structure Local Multiple Alignment of RNA
---
authors: 'Packer, Martin J. and Dauncey, Mark P. and Hunter, Christopher A.'
title: 'Sequence-dependent DNA Structure: Tetranucleotide Conformational Maps'
---
authors: 'Packer, Martin J. and Dauncey, Mark P. and Hunter, Christopher A.'
title: 'Sequence-dependent DNA Structure: Dinucleotide Conformational Maps'
---
authors: 'Pagani, Franco and Baralle, Francisco E.'
title: 'Genomic variants in exons and introns: identifying the splicing spoilers'
---
authors: 'Pagani, Franco and Raponi, Michela and Baralle, Francisco E.'
doi: 10.1073/pnas.0502288102
title: Synonymous mutations in CFTR exon 12 affect splicing and are not neutral in evolution
---
authors: 'Pagani, Franco and Stuani, Cristiana and Zuccato, Elisabetta and Kornblihtt, Alberto R. and Baralle, Francisco E.'
doi: 10.1074/jbc.M209676200
title: Promoter Architecture Modulates CFTR Exon 9 Skipping
---
authors: 'Pai, Raj D. and Oppegard, Lisa M. and Connell, Gregory J.'
doi: 10.1261/rna.2175703
title: Sequence and structural requirements for optimal guide RNA-directed insertional editing within Leishmania tarentolae
---
authors: 'Pál, Csaba and Papp, Balázs and Lercher, Martin J.'
doi: 10.1038/nrg1838
title: An integrated view of protein evolution
---
authors: 'Pánek, Josef and Bobek, Jan and Mikulík, Karel and Basler, Marek and Vohradský, Jiří'
doi: 10.1186/1471-2164-9-217
title: Biocomputational prediction of small non-coding RNAs in Streptomyces
---
authors: 'Antal, Peter and Glenisson, Patrick and Fannes, Geert'
title: On the potential of domain literature for clustering and Bayesian network learning
---
authors: 'Parida, Laxmi and Mishra, Bud'
title: 'Partitioning single-molecule maps into multiple populations: algorithms and probabilistic analysis'
---
authors: 'Park, Britt H. and Levitt, Michael'
title: The Complexity and Accuracy of Discrete State Models of Protein Structure
---
authors: 'Parra, Genis and Blanco, Enrique and Guigo, Roderic'
title: GeneID in Drosophila
---
authors: 'Patel, Abhijit A. and Steitz, Joan A.'
doi: 10.1038/nrm1259
title: 'SPLICING DOUBLE: INSIGHTS FROM THE SECOND SPLICEOSOME'
---
authors: 'Patikoglou, G.'
title: EUKARYOTIC TRANSCRIPTION FACTOR-DNA COMPLEXES
---
authors: 'Pavesi, Giulio and Mauri, Giancarlo'
title: Predicting Conserved Hairpin Motifs in Unaligned RNA Sequences
---
authors: 'Pawson, Tony and Raina, Monica and Nash, Piers'
title: 'Interaction domains: from simple binding events to complex cellular behavior'
---
authors: 'Pawson, Tony'
title: Regulation and targets of receptor tyrosine kinases
---
authors: 'Pawson, Tony'
title: 'Specificity in Signal Transduction: From Phosphotyrosine-SH2 Domain Interactions to Complex Cellular Systems'
---
authors: 'Pearl, Frances M. G. and Lee, David and Bray, James E. and Sillitoe, Ian and Todd, Annabel E. and Harrison, Andrew P. and Thornton, Janet M. and Orengo, Christine A.'
title: Assigning genomic sequences to CATH
---
authors: 'Pedersen, Jakob Skou and Bejerano, Gill and Siepel, Adam and Rosenbloom, Kate and Lindblad-Toh, Kerstin and Lander, Eric S. and Kent, Jim and Miller, Webb and Haussler, David'
doi: 10.1371/journal.pcbi.0020033
title: Identification and Classification of Conserved RNA Secondary Structures in the Human Genome
---
authors: 'Pedersen, Anders Gorm and Baldi, Pierre and Chauvin, Yves and Brunak, Sùren'
title: DNA Structure in Human RNA Polymerase II Promoters
---
authors: 'Peirano, Reto I. and Wegner, Michael'
title: The glial transcription factor Sox10 binds to DNA both as monomer and dimer with different functional consequences
---
authors: 'Peneff, Caroline and Ferrari, Paul and Charrier, Veronique and Taburet, Yvette and Veronique, CeliaMonnier and Zamboni,  and Winter, Jacques and Harnois, Marzia and Fassy, Florence and Bourne, Yves'
title: 'Crystal structures of two human pyrophosphorylase isoforms in complexes with UDPGlc(Gal)NAc: role of the alternatively spliced insert in the enzyme oligomeric assembly and active site architecture'
---
authors: 'Peng, Kang and Radivojac, Predrag and Vucetic, Slobodan and Dunker, A Keith and Obradovic, Zoran'
doi: 10.1186/1471-2105-7-208
title: Length-dependent prediction of protein intrinsic disorder
---
authors: 'Pereira, Manuela S. and Andrade, Lucio and El Difrawy, Sameh and Karger, Barry L. and Manolakos, Elias S.'
title: Statistical learning formulation of the DNA base-calling problem and its solution in a Bayesian EM framework
---
authors: 'Prez-Martn, Jos and de Lorenzo, Vctor'
title: CLUES AND CONSEQUENCES OF DNA BENDING IN TRANSCRIPTION
---
authors: 'Peri, Suraj and Pandey, Akhilesh'
title: A reassessment of the translation initiation codon in vertebrates
---
authors: 'Périer, Rouaïda Cavin and Praz, Viviane and Junier, Thomas and Bonnard, Claude and Bucher, Philipp'
title: The Eukaryotic Promoter Database (EPD)
---
authors: 'Perriquet, O. and Touzet, H. and Dauchet, M.'
title: Finding the common structure shared by two homologous RNAs
---
authors: 'Pertea, Mihaela and Lin, Xiaoying and Salzberg, Steven L.'
title: 'GeneSplicer: a new computational method for splice site prediction'
---
authors: 'Pervouchine, Dmitri D.'
title: 'IRIS: Intermolecular RNA Interaction Search'
---
authors: 'Pesant, Gilles'
title: 'Counting Solutions of CSPs: A Structural Approach'
---
authors: 'Li, Li and Stoeckert, Jr., Christian J. and Roos, David S.'
title: 'OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes'
doi: 10.1101/gr.1224503
---
authors: 'Pevzner, Pavel A. and Tang, Haixu and Waterman, Michael S.'
title: A New Approach to Fragment Assembly in DNA Sequencing
---
authors: 'Pfeiffer, Verena and Papenfort, Kai and Lucchini, Sacha and Hinton, Jay C D and Vogel, Jörg'
doi: 10.1038/nsmb.1631
title: Coding sequence targeting by MicC RNA reveals bacterial mRNA silencing downstream of translational initiation
---
authors: 'Pfeiffer, Verena and Sittka, Alexandra and Tomer, Raju and Tedin, Karsten and Brinkmann, Volker and Vogel, Jörg'
doi: 10.1111/j.1365-2958.2007.05991.x
title: A small non-coding RNA of the invasion gene island (SPI-1) represses outer membrane protein synthesis from the Salmonella core genome
---
authors: 'Pfingsten, Jennifer S. and Kieft, Jeffrey S.'
doi: 10.1261/rna.987808
title: 'RNA structure-based ribosome recruitment: Lessons from the Dicistroviridae intergenic region IRESes'
---
authors: 'Philipps, Dana L. and Park, Jung W. and Graveley, Brenton R.'
doi: 10.1261/rna.7136104
title: A computational and experimental approach toward a priori identification of alternatively spliced exons
---
authors: 'Phillips, Kathryn and Luisi, Ben'
doi: 10.1006/jmbi.2000.4107
title: 'The Virtuoso of Versatility: POU Proteins that Flex to Fit'
---
authors: 'Phizicky, Eric and Bastiaens, Philippe I. H. and Zhu, Heng and Snyder, Michael and Fields, Stanley'
title: Protein analysis on a proteomic scale
---
authors: 'Ferre-DAmare, Adrian R. and Zhou, Kaihong and Doudna, Jennifer A.'
title: 'Crystal structure of a hepatitis delta virus ribozyme'
---
authors: 'Piazza, C. and Antoniotti, M. and Mysore, V. and Policriti, A. and Winkler, F. and Mishra, B.'
title: 'Algorithmic Algebraic Model Checking I: Challenges from Systems Biology'
---
authors: 'Pichon, Christophe and Felden, Brice'
doi: 10.1111/j.1574-6976.2007.00079.x
title: Proteins that interact with bacterial small RNA regulators
---
authors: 'Pickert, L. and Reuter, I. and Klawonn, F. and Wingender, E.'
title: Transcription regulatory region analysis using signal detection and fuzzy clustering
---
authors: 'Pierce, Niles A. and Winfree, Erik'
title: Protein Design is NP-hard
---
authors: 'Pierri, Ciro Leonardo and De Grassi, Anna and Turi, Antonio'
doi: 10.1002/prot.22070
title: Lattices for ab initio protein structure prediction
---
authors: 'Pilon, Marinus and Owen, Jennifer D. and Garifullina, Gulnara F. and Kurihara, Tatsuo and Mihara, Hisaaki and Esaki, Nobuyoshi and Pilon-Smits, Elizabeth A. H.'
doi: 10.1104/pp.102.014639
title: Enhanced Selenium Tolerance and Accumulation in Transgenic Arabidopsis Expressing a Mouse Selenocysteine Lyase
---
authors: 'Pinney, John W. and Rattray, Magnus and Robertson, David L.'
title: Understanding the evolution of protein interaction networks
---
authors: 'Pitulle, C. and Garcia-Paris, M. and Zamudio, K. R. and Pace, N. R.'
title: Comparative structure analysis of vertebrate ribonuclease P RNA
---
authors: 'Plass, Mireya and Eyras, Eduardo'
doi: 10.1186/1471-2148-6-50
title: Differentiated evolutionary rates in alternative exons and the implications for splicing regulation
---
authors: 'Plasterk, Ronald HA and Ketting, René F'
title: The silence of the genes
---
authors: 'Pollard, Katherine S. and Salama, Sofie R. and King, Bryan and Kern, Andrew D. and Dreszer, Tim and Katzman, Sol and Siepel, Adam and Pedersen, Jakob S. and Bejerano, Gill and Baertsch, Robert and Rosenbloom, Kate R. and Kent, Jim and Haussler, David'
doi: 10.1371/journal.pgen.0020168
title: Forces Shaping the Fastest Evolving Regions in the Human Genome
---
authors: 'Pollard, Katherine S. and Salama, Sofie R. and Lambert, Nelle and Lambot, Marie-Alexandra and Coppens, Sandra and Pedersen, Jakob S. and Katzman, Sol and King, Bryan and Onodera, Courtney and Siepel, Adam and Kern, Andrew D. and Dehay, Colette and Igel, Haller and Ares, Jr., Manuel and Vanderhaeghen, Pierre and Haussler, David'
doi: 10.1038/nature05113
title: An RNA gene expressed during cortical development evolved rapidly in humans
---
authors: 'Ponomarenko, Julia V. and Orlova, Galina V. and Frolov, Anatoly S. and Gelfand, Mikhail S. and Ponomarenko, Mikhail P.'
title: 'SELEX_DB: a database on in vitro selected oligomers adapted for recognizing natural sites and for analyzing both SNPs and site-directed mutagenesis data'
---
authors: 'Ponomarenko, Julia V. and Ponomarenko, Mikhail P. and Frolov, Anatoly S. and Vorobyev, Denis G. and Overton, G. Christian and Kolchanov, Nikolay A.'
title: Conformational and physicochemical DNA features specific for transcription factor binding sites
---
authors: 'Ponomarenko, M. P. and Ponomarenko, J. V. and Kel, A. E. and Kolchanov, N. A.'
title: SEARCH FOR DNA CONFORMATIONAL FEATURES FOR FUNCTIONAL SITES. INVESTIGATION OF THE TATA BOX
---
authors: 'Pozzoli, U. and Sironi, M.'
doi: 10.1007/s00018-005-5030-6
title: Silencers regulate both constitutive and alternative splicing events in mammals
---
authors: 'Prakash, Amol and Tompa, Martin'
doi: 10.1038/nbt1140
title: Discovery of regulatory elements in vertebrates through comparative genomics
---
authors: 'Praz, Viviane and Périer, Rouaïda and Bonnard, Claude and Bucher, Philipp'
title: 'The Eukaryotic Promoter Database, EPD: new entry types and links to gene expression data'
---
authors: 'Preger, Ella and Ziv, Inbal and Shabtay, Ariel and Sher, Ifat and Tsang, Michael and Dawid, Igor B. and Altuvia, Yael and Ron, Dina'
doi: 10.1073/pnas.0307952100
title: Alternative splicing generates an isoform of the human Sef gene with altered subcellular localization and specificity
---
authors: 'Prestwich, Steven'
title: A Local Search Algorithm for Balanced Incomplete Block Designs
---
authors: 'Prestwich, Steven and Higgins, Des and OSullivan, Orla'
title: A SAT-Based Approach to Multiple Sequence Alignment
---
authors: 'Prévost, Karine and Salvail, Hubert and Desnoyers, Guillaume and Jacques, Jean-François and Phaneuf, Émilie and Massé, Eric'
doi: 10.1111/j.1365-2958.2007.05733.x
title: 'The small RNA RyhB activates the translation of shiA mRNA encoding a permease of shikimate, a compound involved in siderophore synthesis'
---
authors: 'Profit, Adam A. and Lee, Tae Ryong and Niu, Jinkui and Lawrence, David S.'
doi: 10.1074/jbc.M009262200
title: 'Molecular Rulers: An Assessment of Distance and Spatial Relationships of Src Tyrosine Kinase SH2 and Active Site Regions'
---
authors: 'Prohaska, Sonja and Fried, Claudia and Flamm, Christoph and Wagner, Gnter P. and Stadler, Peter F.'
title: 'Surveying Phylogenetic Footprints in Large Gene Clusters: Applications to Hox Cluster Duplications'
---
authors: 'Proudfoot, Nick J. and Furger, Andre and Dye, Michael J.'
title: Integrating mRNA Processing with Transcription
---
authors: 'Prusinkiewicz, Przemyslaw'
title: Modeling plant growth and development
---
authors: 'Pudimat, Rainer'
title: Probabilistische Modelle zur Beschreibung und Vorhersage regulativer DNA-Sequenzen
---
authors: 'Puget, Jean-Fran¸ois'
title: Symmetry Breaking Revisited
---
authors: 'Pulvermacher, Sarah C. and Stauffer, Lorraine T. and Stauffer, George V.'
doi: 10.1099/mic.0.023432-0
title: Role of the Escherichia coli Hfq protein in GcvB regulation of oppA and dppA mRNAs
---
authors: 'Pulvermacher, Sarah C. and Stauffer, Lorraine T. and Stauffer, George V.'
doi: 10.1128/JB.00915-08
title: The Small RNA GcvB Regulates sstT mRNA Expression in Escherichia coli
---
authors: 'Putland, R. A. and Sassinis, T. A. and Harvey, J. S. and Diamond, P. and Coles, L. S. and Brown, C. Y. and Goodall, G. J.'
doi: 10.1128/MCB.22.6.1664-1673.2002
title: RNA Destabilization by the Granulocyte Colony-Stimulating Factor Stem-Loop Destabilizing Element Involves a Single Stem-Loop That Promotes Deadenylation
